### PR TITLE
Adding Week 9 Blogpost

### PR DESCRIPTION
**Pull request by ganimtron-10**
Adding Weekly Blogpost

msedge_6sB1Wv5jAC

### CODE REVIEW

File: docs/source/posts/2022/2022-08-03-week-9-praneeth.rst [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+.. post:: August 3 2022
+   :author: Praneeth Shetty 
+   :tags: google
+   :category: gsoc
+
+
+What did you do this week?
+--------------------------
+I started this week by creating a quick PR `#645 <https://github.com/fury-gl/fury/pull/645>`_ for the UI sliders. The sliders raised ``ZeroDivsionError`` when the min and max values were the same. To solve this I handled the case where the value_range becomes zero and then manually set the handle position to zero.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-03-week-9-praneeth.rst:13]: missing a comma after `To solve this`

File: docs/source/posts/2022/2022-08-03-week-9-praneeth.rst [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+Then I updated the implementation of the Grouping Shapes feature to support the translation and rotation of the shapes grouped together as you can see below.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-03-week-9-praneeth.rst:15]: Missing a comma after `grouped together`

File: docs/source/posts/2022/2022-08-03-week-9-praneeth.rst [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    :width: 400
+    :align: center
+
+Did you get stuck anywhere?
+---------------------------
+No, Everything went well.
+
+What is coming up next?
+-----------------------
+Completing pending PRs and searching various approaches to implement the freehand drawing.
```
Comment (m-agour) [inline docs/source/posts/2022/2022-08-03-week-9-praneeth.rst:33]: Maybe `Complete the pending PRs` of something like this.

**Review by m-agour — COMMENTED** [review]
Hello @ganimtron-10,
Please see my comments below
Thanks

File: docs/source/posts/2022/2022-08-03-week-9-praneeth.rst [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+After this, I worked on the `PolyLine` and removed the extra point being added to the start of the `PolyLine` whenever a new instance was created.
+
+.. image:: https://user-images.githubusercontent.com/64432063/183280803-5d7ae350-f080-478d-8a2f-a71460037ea4.gif
+    :width: 400
+    :align: center
+
+Did you get stuck anywhere?
+---------------------------
+No, Everything went well.
```
Comment (skoudoro) [inline docs/source/posts/2022/2022-08-03-week-9-praneeth.rst:29]: `everything` smaller case

**Review by skoudoro — COMMENTED** [review]
where is your review @xtanion?

@ganimtron-10, can you address @m-agour comments?

File: docs/source/posts/2022/2022-08-03-week-9-praneeth.rst (hunk 3 above)
Comment (xtanion) [inline docs/source/posts/2022/2022-08-03-week-9-praneeth.rst:33]: Maybe break the sentence into two parts.
`for implementing the freehand drawing` maybe

**Review by xtanion — COMMENTED** [review]
Nice blog @ganimtron-10 , PTAL at my comment  below

**Review by skoudoro — APPROVED** [review]
All good, thanks @ganimtron-10, and for all the reviews. Merging!

**Pull request merged**